students/dongmyeong/users/views.py

```diff
@@ -18,7 +21,9 @@ def post(self, request):
                     not check_phone_validation(data['phone']):
                         raise ValidationError("VALIDATION_ERROR")
 
-            User.objects.create(email=data['email'], password=data['password'], phone=data['phone'], nickname=data['nickname'])
+            hashed_pw = bcrypt.hashpw(data['password'].encode('utf-8'), bcrypt.gensalt())
+            User.objects.create(email=data['email'], password=hashed_pw.decode('utf-8'), phone=data['phone'], nickname=data['nickname'])
```

Comment:
```suggestion
            User.objects.create(
                 email    = data['email'], 
                 password = hashed_pw.decode('utf-8'), 
                 phone    = data['phone'], 
                 nickname = data['nickname']
             )
```

적절히 줄바꿈 해주는 것도 가독성을 높이는 좋은 방법입니다!

Reply:
수정 완료하였습니다.